main.css

```diff
@@ -2,10 +2,13 @@
   margin: 0;
   padding: 0;
   border: 0;
-  background-color: black;
   color: white;
 }
 
+body {
+  background-color: black;
+}
+
```

Comment:
Pretty much do the same thing, but this is convention

Reply:
I see, it would be better to have it in it's own body styling.